docs/standard/microservices-architecture/microservice-ddd-cqrs-patterns/infrastructure-persistence-layer-design.md

```diff
@@ -3,8 +3,9 @@ title: Designing the infrastructure persistence layer
 description: .NET Microservices Architecture for Containerized .NET Applications | Designing the infrastructure persistence layer
 author: CESARDELATORRE
 ms.author: wiwagn
-ms.date: 11/08/2017
+ms.date: 06/11/2017
 ---
+
 # Designing the infrastructure persistence layer
 
 Data persistence components provide access to the data hosted within the boundaries of a microservice (that is, a microservice’s database). They contain the actual implementation of components such as repositories and [Unit of Work](https://martinfowler.com/eaaCatalog/unitOfWork.html) classes, like custom EF DBContexts.
```

Comment:
One thing I'm struggling with this topic is to define what's the purpose of it as I start reading it. There is no real introduction/summary and why you would cover the different patterns.